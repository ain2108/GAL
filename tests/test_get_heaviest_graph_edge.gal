int main()
{
	/*
	A - 1
	B - 2
	C - 3
	D - 4
	E - 5
	F - 6
	*/

	node n1;
    n1 = |"A": 7, "B", 14, "F", 9, "C"|; 
    node n2;
    n2 = |"B": 7, "A", 10, "C"|;
    node n3; 
    n3 = |"C": 5, "D", 16, "E"|; 
    node n4;
    n4 = |"D": 20, "A", 7, "B"|; 

    elist n5;
    n5 = [|"E", 24, "D"|::|"E", 13, "B"|];

    nlist graph;
    graph = [n1::n2::n3::n4::n5];  

    edge temp;

    temp = get_heaviest_graph_edge(graph);

    print_edge(temp);

    return 1;

}