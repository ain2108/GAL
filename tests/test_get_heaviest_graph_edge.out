|E, 24, D|